geomesa-compute/pom.xml

```diff
@@ -41,6 +41,16 @@
             <scope>compile</scope>
         </dependency>
         <dependency>
+            <groupId>org.locationtech.geomesa</groupId>
+            <artifactId>geomesa-jobs-accumulo1.5</artifactId>
+            <exclusions>
```

Comment:
It's the same pattern that @elahrvivaz used in a couple other places, I think. .... I think?